src/sentry/api/serializers/models/team.py

```diff
@@ -70,24 +71,30 @@ def serialize(self, obj, attrs, user):
 
 class TeamWithProjectsSerializer(TeamSerializer):
     def get_attrs(self, item_list, user):
-        project_qs = list(
-            Project.objects.filter(
+        project_team_qs = list(
```

Comment:
nit: This is named as `_qs` implying it's a QuerSet, and it's just a list. I know this preexisted, but might as well change it since you already did.